java/client/src/org/openqa/selenium/chrome/ChromeDriver.java

```diff
@@ -207,4 +210,9 @@ public void setLocation(Location location) {
   public void launchApp(String id) {
     execute(ChromeDriverCommand.LAUNCH_APP, ImmutableMap.of("id", id));
   }
+
+  @Override
+  public TouchScreen getTouch() {
+	return new RemoteTouchScreen(this.getExecuteMethod());
```

Comment:
would be better to store as a local private variable (like locationContext / webStorage) and only inialize it once on construction.
also fix the indentation here